site/src/components/InputGroup/InputGroup.tsx

```diff
@@ -0,0 +1,55 @@
+import type { FC, HTMLProps } from "react";
+
+export const InputGroup: FC<HTMLProps<HTMLDivElement>> = (props) => {
+  return (
+    <div
+      {...props}
+      css={{
+        "&": {
+          display: "flex",
+          alignItems: "flex-start",
+        },
+
+        "& > *:hover": {
+          zIndex: 1,
+        },
```

Comment:
I'm not sure what the z Index is doing here. Is this to handle some MUI styling?

Reply:
Oh, unless it's specifically to create a stacking context, but in that case, I feel like a comment would be good